recipes/liburing/all/conanfile.py

```diff
@@ -66,6 +71,9 @@ def generate(self):
         tc = AutotoolsToolchain(self)
         tc.update_configure_args({
             "--nolibc": None if self.options.get_safe("with_libc", default=True) else "",
+            "--use-libc": "" if self.options.get_safe("use_libc", default=False) else None,
```

Comment:
And I think this would have to be extracted to (untested):
```py
if Version(self.version) >= "2.5":
    if self.options.with_libc:
        tc.configure_args.append("--use-libc")
elif Version(self.version >= "2.2":
    if not self.options.with_libc:
        tc.configure_args.append("--nolibc")
```

Reply:
done